common/src/main/java/io/seata/common/exception/EurekaRegistryException.java

```diff
@@ -54,6 +54,6 @@ public EurekaRegistryException(String message, Throwable cause) {
      * @param cause the cause
      */
     public EurekaRegistryException(Throwable cause) {
-        super(cause);
+        super(cause.getMessage(), cause);
```

Comment:
Please read the implementation in super (cause)

Reply:
resolved